contents/docs/experiments/analyzing-results.mdx

```diff
@@ -92,6 +94,33 @@ Click on any variant in the delta chart to access detailed results. This opens a
 
 Click **View recordings** to see session recordings of experiment participants. By default, all metric events are applied as filters to the playlist meaning you'll only see recordings for users who completed the funnel or fired the trend metric. However, the filters don't map exactly to the statistical calculations ([funnel attribution type](/docs/product-analytics/funnels#attribution-types) isn't applied, for instance), so the recordings don't map exactly to the experiment results.
 
+### Time series analysis (beta)
+
+The time series chart shows how your experiment results change over time, helping you see if your results are stable or still changing:
+
+<ProductScreenshot
+    imageLight={TimeSeriesLight} 
+    imageDark={TimeSeriesDark}
+    alt="Time series chart showing experiment results over time with confidence intervals" 
+    classes="rounded"
+/>
+
+How the chart works:
+
+- **Center line**: Shows the percentage change between your variants day by day
+- **Shaded bounds**: The confidence interval that gets narrower as you collect more data
+- **Colored areas**: 
+  - Green means the variant is winning with statistical significance
```

Comment:
Do we document that we can define what is a win/lose somewhere we can link? I think it can be slightly confusing now because it's not just big number up and to the right. Some link here might help

Reply:
Nice